scripts/api_review/template.md

```diff
@@ -1,36 +1,45 @@
+# {name}
+
 - start = 2021-01-23 18:10
 - end = 2021-01-23 18:10
-- reviewer = Stefan Hanreich <stefanhani@gmail.com>
-
+- moderator = Stefan Hanreich <stefanhani@gmail.com>
 
-## Checklist
 
-### Documentation
+## Signature
+{signature}
```

Comment:
Sometimes there are newlines after the headings, sometimes not. Did you use the `scripts/dev/reformat-markdown` script?

Reply:
didn't know this existed - will do!

Reply:
ran it now - doesnt seem to have changed anything. not sure if my md is included.
I have adjusted the newlines manually for now though.